SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1447,71 +1448,76 @@ USpatialNetConnection* USpatialNetDriver::AcceptNewPlayer(const FURL& InUrl, FUn
 
 	FString ErrorMsg;
 	AGameModeBase* GameMode = GetWorld()->GetAuthGameMode();
```

Comment:
Any idea why we're using GetWorld() and World interchangeably?

Reply:
No.  outside of PostSpawnPlayerController this is largely code that i've shifted around.  I'll do a cleanup pass through it